src/main/java/org/junitpioneer/jupiter/DisableIfDisplayName.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2015-2020 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v2.0 which
+ * accompanies this distribution and is available at
+ *
+ * http://www.eclipse.org/legal/epl-v20.html
+ */
+
+package org.junitpioneer.jupiter;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.condition.DisabledIf;
+import org.junit.jupiter.api.extension.ExecutionCondition;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.api.extension.ExtensionContext;
+import org.junit.jupiter.params.ParameterizedTest;
+
+/**
+ * {@code @DisableIfDisplayName} is a JUnit Jupiter extension which can be used to
+ * selectively disable {@link ParameterizedTest} based on their {@link ExtensionContext#getDisplayName()}
+ *
+ * <p>
+ * The extension is an {@link ExecutionCondition} which validates dynamically registered tests
+ * This is highly useful since current {@link Disabled} or {@link DisabledIf} annotations disable
+ * the whole test but not the Parameterized tests selectively
```

Comment:
```suggestion
 * This is highly useful since current {@link Disabled} or {@link DisabledIf} annotations disable
 * the whole test, but not the Parameterized tests selectively.
```